# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1345572207 14400
#      Tue Aug 21 14:03:27 2012 -0400
# Node ID 26115efd659233580559590a8aa1cbd14a517f40
# Parent  a3c8a3c2dbdad5691ed58e5069c6ac974584b2cf
link liboctgui with liboctinterp.la

* libgui/src/Makefile.am (liboctgui_la_LIBADD): Include
../../libinterp/liboctinterp.la in the list.

diff --git a/libgui/src/Makefile.am b/libgui/src/Makefile.am
--- a/libgui/src/Makefile.am
+++ b/libgui/src/Makefile.am
@@ -113,16 +113,17 @@ liboctgui_la_CPPFLAGS = \
   -I$(srcdir)/../../libinterp/octave-value \
   -I$(srcdir)/../../libtinerp/operators \
   -I$(srcdir)/../../libinterp/parse-tree
 
 include link-deps.mk
 
 liboctgui_la_LIBADD = \
   libqterminal.la \
+  ../../libinterp/liboctinterp.la \
   ../../liboctave/liboctave.la \
   ../../libcruft/libcruft.la \
   $(LIBOCTGUI_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
 liboctgui_current = 0
 liboctgui_revision = 0
 liboctgui_age = 0
